src/config/rcxml.c

```diff
@@ -235,6 +354,13 @@ xml_tree_walk(xmlNode *node)
 			traverse(n);
 			in_keybind = false;
 			continue;
+		} 
+		if(!strcasecmp((char *)n->name, "mousebind")) {
+			in_mousebind = true;
+			traverse(n);
+			in_mousebind = false;
+			add_new_mousebinds();
```

Comment:
Great progress. I think we’re very close.
We should not add_new_mousebind here.
I’d need to play with it myself to check, but think that mousebind_create() should be called from fill_mousebind() the first time we get to a new ‘<mousebind>’ element - and after truncation I’m guessing that nodename would then be empty (“”).
If it runs fine, I’m happy to merge now and we can tweak/re-factor a bit when I’m back behind a machine.
I’m not that pedantic on coding style, but if you’re happy to fix the following, that’d be great
- Space between if and opening bracket
- When declaring pointers, but the star with the binding (so do ‘struct a *b’ rather than ‘struct a* b’.
Do you want me to quash the commits when merging?

Reply:
I try to keep consistent with this https://git.sr.ht/~sircmpwn/cstyle which is pretty close to wlroots/sway.

Have no clang-format.

Just say if you prefer me to do anything. I’m grateful for support, but no pressure.